packages/template-retail-react-app/app/components/_error/index.jsx

```diff
@@ -52,7 +50,7 @@ const Error = (props) => {
                         icon={<BrandLogo width={[8, 8, 8, 12]} height={[6, 6, 6, 8]} />}
                         marginBottom={[1, 1, 2, 0]}
                         variant="unstyled"
-                        onClick={() => history.push('/')}
+                        onClick={() => (window.location.href = '/')}
```

Comment:
So that I understand clearly -- the idea here is that the application is in an error state from a JavaScript perspective, so in some sense "React has crashed" so the `window.history` listener in React doesn't pick up the change so we just hard navigate?

Reply:
Yes, that's correct. If we use history, we trigger a soft navigation and are still in an error state so the home page is not rendered. We need to trigger a hard navigate to get out of the error state.